jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/FrameHandler.java

```diff
@@ -141,13 +141,17 @@ default boolean isDemanding()
          */
         Duration getIdleTimeout();
 
+        Duration getWriteIdleTimeout();
+
         /**
          * Set the Idle Timeout.
          *
          * @param timeout the timeout duration
          */
         void setIdleTimeout(Duration timeout);
 
+        void setWriteIdleTimeout(Duration timeout);
```

Comment:
what does -1 and 0 timeout mean?   Can you also fix javadoc for `setIdleTimeout` to explain -1 and/or 0 values?

Reply:
there is currently no special meaning for zero or negative timeouts, it just does not use a timeout